libs/cart/src/effects/cart-item.effects.ts

```diff
@@ -65,29 +72,45 @@ export class DaffCartItemEffects<
   add$ = this.actions$.pipe(
     ofType(DaffCartItemActionTypes.CartItemAddAction),
     switchMap((action: DaffCartItemAdd<U>) =>
-      this.driver.add(
-        this.storage.getCartId(),
-        action.input
-      ).pipe(
-        map((resp: V) => new DaffCartItemAddSuccess(resp)),
+      combineLatest([
+				this.facade.items$.pipe(
```

Comment:
This facade should likely NOT be used here, I think **generally** facade usage in an effect is an antipattern.

Reply:
The reference to state here isn't needed anymore.